CodeStarterCamp_Week3/CafePosMachine.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CoffeePosMachine.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by minsong kim on 2023/03/06.
+//
+
+import Foundation
+
+class Person {
+    var money: Int
+    var nickname: String
+    var myOrder: Coffee?
+    
+    init(money: Int, nickname: String) {
+        self.money = money
+        self.nickname = nickname
+    }
+    
+    func order(_ coffee: Coffee) {
+        myOrder = coffee
+    }
+}
+
+enum Coffee: String {
+    case americano = "아메리카노", decaffeine = "디카페인"
+    case latte = "라떼", vanilla = "바닐라라떼", caramel = "카라멜 마끼아또", cappuccino = "카푸치노"
+    case nothing = ""
+}
+
+struct CoffeeShop {
+    var barista: Person
+    var totalSales: Int
+    var pickUpTable: Array<String>
+    var menu: [Coffee: Int?] = [Coffee.americano: 2500, Coffee.decaffeine: 2500, Coffee.vanilla: 3500, Coffee.caramel: 3500, Coffee.cappuccino: 3500]
+        
+    mutating func takeOrder(from name: Person){
+        let customerOrder: Coffee? = name.myOrder
+        let price: Int? = self.menu[name.myOrder ?? .nothing] ?? 0
+        if let customerOrder = customerOrder {
```

Comment:
메서드 내에서도 가독성을 위해 선언부와 실행부를 개행으로 나눠주는 것이 좋습니다 ㅎㅎ
```suggestion
        let customerOrder: Coffee? = name.myOrder
        let price: Int? = self.menu[name.myOrder ?? .nothing] ?? 0
        
        if let customerOrder = customerOrder {
```